debian/changelog

```diff
@@ -1,6 +1,7 @@
 kano-updater (2.3.0-1) unstable; urgency=low
 
   * Install Make Light by default
+  * Improved Flappy Judoka game launching
```

Comment:
As this is being released for `v2.4.0`, you're going to need to bump the version here.